### PR TITLE
fix(files): Ignore right click for default action

### PR DESCRIPTION
**Pull request by artonge**
Follow up of https://github.com/nextcloud/server/pull/45571

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

File: .htaccess [hunk 1]
```diff
@@ -108,3 +108,7 @@
 
 AddDefaultCharset utf-8
 Options -Indexes
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (susnux) [inline .htaccess:111]: This should not be committed

File: apps/files/src/components/FileEntryMixin.ts [hunk 2]
```diff
@@ -208,6 +211,11 @@ export default defineComponent({
 		},
 
 		execDefaultAction(event) {
+			// Ignore right click.
+			if (event.button === 2) {
```
Comment (susnux) [inline apps/files/src/components/FileEntryMixin.ts:215]: Should we ignore all non-main buttons? I mean if I press my tertiary button I would not expect the action to be performed.
The only button I am not sure about is middle-mouse button for opening in new tab?

```suggestion
			if (event.button != 2) {
```

File: .htaccess (hunk 1 above)
Comment (artonge) [reply .htaccess:111]: Arg, thanks

File: apps/files/src/components/FileEntryMixin.ts (hunk 2 above)
Comment (artonge) [reply apps/files/src/components/FileEntryMixin.ts:215]: From what I understand,
0 is left
1 is middle
2 is right
3 and above is whatever

`if (event.button >= 1) {` ?

File: apps/files/src/components/FileEntryMixin.ts [hunk 3]
```diff
@@ -208,6 +208,11 @@ export default defineComponent({
 		},
 
 		execDefaultAction(event) {
+			// Ignore right click.
+			if (event.button > 1) {
+				retrurn
```
Comment (nickvergessen) [inline apps/files/src/components/FileEntryMixin.ts:213]:
```suggestion
				return
```

**Review by nickvergessen — DISMISSED** [review]

File: apps/files/src/components/FileEntryMixin.ts (hunk 3 above)
Comment (artonge) [reply apps/files/src/components/FileEntryMixin.ts:213]: :face_exhaling:

**Review by blizzz — APPROVED** [review]

File: apps/files/src/components/FileEntryMixin.ts (hunk 2 above)
Comment (susnux) [reply apps/files/src/components/FileEntryMixin.ts]: Makes sense

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**